.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
do we usually commit this file?

Reply:
agree we don't. we can update the .gitignore

Reply:
Done